streaming-payments/programs/streaming-payments/src/verifying_key_streaming_payments.rs

```diff
@@ -31,13 +31,13 @@ pub const VERIFYINGKEY_PSP_PAYMENT_STREAMING: Groth16Verifyingkey = Groth16Verif
     ],
 
     vk_delta_g2: [
-        3, 217, 119, 153, 126, 240, 200, 176, 234, 231, 196, 249, 226, 153, 122, 175, 212, 132,
-        186, 193, 68, 160, 65, 37, 48, 137, 171, 185, 115, 15, 210, 164, 29, 245, 181, 57, 54, 232,
-        85, 163, 41, 218, 56, 1, 24, 61, 209, 70, 217, 218, 197, 43, 105, 16, 211, 103, 225, 97,
-        243, 112, 12, 108, 124, 45, 45, 157, 233, 139, 200, 209, 242, 125, 102, 166, 246, 28, 165,
-        212, 77, 191, 157, 1, 130, 233, 160, 42, 121, 24, 155, 31, 166, 39, 29, 196, 143, 125, 22,
-        61, 87, 43, 120, 31, 100, 113, 62, 212, 134, 102, 149, 89, 245, 173, 70, 121, 81, 133, 137,
-        68, 86, 7, 1, 103, 35, 175, 234, 42, 11, 240,
+        13, 144, 221, 165, 105, 204, 117, 159, 207, 122, 75, 183, 181, 211, 206, 90, 102, 31, 156,
```

Comment:
> why is the verifyingkey updated but the proving key in build-circuits is not?

Because we already ignore build-circuits. 
[I added **/verifying_key_*.rs to .gitignore ](https://github.com/Lightprotocol/psp-examples/pull/8/commits/528a968d54be68dc3c51c916018ba13b905b0c19)